.github/workflows/ci.yml

```diff
@@ -56,11 +56,11 @@ jobs:
       - name: Maven Checks
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          $RETRY $MAVEN install -B --strict-checksums -V -T C1 -DskipTests -P ci -pl '!:trino-server-rpm'
+          $RETRY $MAVEN install -B --strict-checksums -V -T C1 -DskipTests -P ci -Dgib.disable -pl '!:trino-server-rpm'
       - name: Test Server RPM
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          $RETRY bash -c "$MAVEN verify -B --strict-checksums -P ci -pl :trino-server-rpm || find core/trino-server-rpm/ -exec ls -ald {} +"
+          $RETRY bash -c "$MAVEN verify -B --strict-checksums -P ci -Dgib.disable -pl :trino-server-rpm || find core/trino-server-rpm/ -exec ls -ald {} +"
```

Comment:
What `gib.disable` does? Will GIB be enabled on CI only, or for engineers too?
(for now, i'd prefer CI only)

Reply:
GIB usage needs to be enabled explicitly, by using the `ci` profile, but it'll work just fine when enabled locally by engineers. I disable GIB in this job because it needs to build `:trino-server` which requires all modules to be installed to build the tarball, and then the RPM.

Reply:
Added!